
  * modules/loggers/mod_log_config.c (log_request_time): Log
  the minutes component of the timezone correctly.
                                                                                                  
  PR: 23642
  Submitted by: Hong-Gunn Chew <hgbug@gunnet.org>

--- httpd-2.0.46/modules/loggers/mod_log_config.c.logtimez
+++ httpd-2.0.46/modules/loggers/mod_log_config.c
@@ -570,7 +570,7 @@
                          "[%02d/%s/%d:%02d:%02d:%02d %c%.2d%.2d]",
                          xt.tm_mday, apr_month_snames[xt.tm_mon],
                          xt.tm_year+1900, xt.tm_hour, xt.tm_min, xt.tm_sec,
-                         sign, timz / (60*60), timz % (60*60));
+                         sign, timz / (60*60), (timz % (60*60)) / 60);
             cached_time->t_validate = t_seconds;
             memcpy(&(request_time_cache[i]), cached_time,
                    sizeof(*cached_time));
